packages/liveblocks-core/src/room.ts

```diff
@@ -1837,9 +1873,15 @@ function makeStateMachine<
     const storageOps = state.buffer.storageOperations;
 
     if (storageOps.length > 0) {
+      const hasAlreadyPendingModifications = hasPendingStorageModifications();
       storageOps.forEach((op) => {
         state.offlineOperations.set(nn(op.opId), op);
       });
+      if (!hasAlreadyPendingModifications) {
+        eventHub.pendingStorageModifications.notify({
+          hasPendingStorageModifications: true,
+        });
```

Comment:
@GuillaumeSalles EventSources don't require a payload for an event. If in this case the payload isn't useful (i.e. always true), you may as well just not send a payload at all and just make it a `void` event source, and call `.notify()` without arguments. Nothing wrong with that. Or would it be useful to send an event whenever this boolean value "flips", i.e. also send an event when it changes to `false`?